config/routes.rb

```diff
@@ -1,3 +1,14 @@
 Rails.application.routes.draw do
   resources :events
+
+  constraints Clearance::Constraints::SignedIn.new do
+    root to: 'dashboards#show', as: :signed_in_root
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.